tests/test_sensor.py

```diff
@@ -347,16 +348,21 @@ def test_creation(self, agg_sensor, ss, sensors):
 
```

Comment:
Could add an assertion to `test_creation` that `update_aggregate` was called with `None, None, None` (the current test does check that it was called, but doesn't validate the arguments).